src/nativeapm/nativeapm.ts

```diff
@@ -204,7 +205,7 @@ module ProcessOut {
           this.markdownLibraryInstance,
           this.theme,
           this.capturePayment.bind(this),
-          ""
+          this.gatewayConfiguration.native_apm.parameter_values?.customer_action_message || ""
```

Comment:
I'm 80% sure that because of optional chaining E2E tests will fail in checkout-cdn 😅 Unfortunately PO.js doesn't support new features of JS, in the future it would be good to upgrade TS config, but for now just use:
`this.gatewayConfiguration.native_apm.parameter_values ? this.gatewayConfiguration.native_apm.parameter_values.customer_action_message : ""`

Reply:
Ahh good spot, I had something like this initially but it did not handle case when there is no `customer_action_message` (then the message would end up as undefined instead of empty string) which can happen, so the options are:
```ts
this.gatewayConfiguration.native_apm.parameter_values ? 
this.gatewayConfiguration.native_apm.parameter_values.customer_action_message || "" : ""
```
or
```ts
(this.gatewayConfiguration.native_apm.parameter_values && 
this.gatewayConfiguration.native_apm.parameter_values.customer_action_message) || ""
```

Will go with the second one as it is a bit simpler

Reply:
Done in [9680f99](https://github.com/processout/processout.js/pull/139/commits/9680f99077d44a9bc20dc94bc3f593a6e5e93f3c)

Reply:
Awesome, sounds good